### ISSUE TITLE
Error: Cannot find module '@babel/runtime/core-js/object/keys'

### ISSUE THREAD

**Issue by rollecode** [jaebradley/wakatime-cli#41]
Hello. Just installed successfully but when running `waka setup` I get this:

```` bash
internal/modules/cjs/loader.js:605
    throw err;
    ^

Error: Cannot find module '@babel/runtime/core-js/object/keys'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:603:15)
    at Function.Module._load (internal/modules/cjs/loader.js:529:25)
    at Module.require (internal/modules/cjs/loader.js:657:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/@jaebradley/wakatime-cli/build/services/formatAggregatedStatistics.js:1:270)
    at Module._compile (internal/modules/cjs/loader.js:721:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:732:10)
    at Module.load (internal/modules/cjs/loader.js:620:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:560:12)
    at Function.Module._load (internal/modules/cjs/loader.js:552:3)
````

**Comment by jaebradley** [comment]
@ronilaukkarinen thanks for opening this issue and sorry you were running into problems getting setup!

I just published `1.1.6` which incorporates changes from #43. This includes updating the CLI's `@babel/runtime` dependency.

If you install `1.1.6` do you still run into issues?

If so, do you mind posting your `npm` and `node` versions?

If you're all set, feel free to close this issue.

**Comment by rollecode** [comment]
Yay, it works now! Thanks!
